ESP3Packet.js

```diff
@@ -1,44 +1,5 @@
-/* jslint node: true */
 'use strict'
 
-/**
- * ESP3 packet structure through the serial port.
- *
- * Sync  is 0x55
- * CRC8H is CRC8 for ‘Header’
- * CRC8D is CRC8 for ‘Datas’ (Data and Optional Data)
- *
- *                   bytes
- *                     ▼
- * +---------------+ -----
- * |   Sync Byte   |   1
- * +---------------+ - + - ----------------------+
- * |               |   2    Data length          |
- * |               |   +                         |
- * |    Header     |   1    Optional Data length |
- * |               |   +                         |
- * |               |   1    Packet type          |
- * +---------------+ - + - ----------------------+
- * |  CRC8 Header  |   1
- * +---------------+ - + -
- * |               |
- * |               |
- * |     Data      |   X
- * |               |
- * |               |
- * +---------------+ - + -
- * |               |
- * |               |
- * | Optional Data |   Y
- * |               |
- * |               |
- * +---------------+ - + -
- * |   CRC8 Data   |   1
- * +---------------+ -----
- *
- * also full ESP3 Packet Length = 7 + X(Data length) + Y(Optional Data length)
- */
-
 class ESP3Packet {
   constructor () {
```

Comment:
We can save some lines on concrete packets by using the construct from #36 

https://github.com/RafaelKa/node-serialport-enocean-parser/pull/36/files#diff-c6ea59fd55a6999b6c8b3838f6146175R8

 see ESP3/Packet.js and PacketTypes/RADIO_ERP1.js from the branch

Reply:
can we land this, and do optimizations and improvements in separat PRs?

Reply:
and please don't freeze the objects. there is no benefit in doing so, and in turn takes away some flexibility for users of this module...